modules/nodes-pods-autoscaling-policies.adoc

```diff
@@ -0,0 +1,92 @@
+
+By default, the scaling policies are implemented when the number of Pods associated with an HPA is greater than 40.
+
+.Sample HPA object with a scaling policy
+[source, yaml]
+----
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: hpa-resource-metrics-memory
+  namespace: default
+spec:
+  behavior:
+    scaleDown:
+      policies: <1>
+      - type: Pods <2>
+        value: 4 <3>
+        periodSeconds: 60 <4>
+      - type: Percent
+        value: 10
+        periodSeconds: 60
+      selectPolicy: Min <5>
+      stabilizationWindowSeconds: 300 <6>
+    scaleUp:
+      policies:
+      - type: Pods
+        value: 5
+        periodSeconds: 70
+      - type: Percent
+        value: 12
+        periodSeconds: 80
+      selectPolicy: Max
+      stabilizationWindowSeconds: 0
+...
+----
+
+<1> Scaling policies allows you to control autoscaling by the number of pods or a percentage over a period of seconds for scaling up or down. 
+<2> The `type` parameter determines if the policy scales by a specific number of Pods or a percentage of Pods during each iteration. The default value is `Pods`.
+<3> The `value` parameter determines the amount of scaling, either the number of Pods or percentage of Pods during each iteration. The default value is `4`.
+<4> The `periodSeconds` parameter determines the length of a scaling iteration. The default value is `15` seconds.
+<5> The `selectPolicy` parameter determines which policy to use first, if multiple policies are defined. Specify `Max` to use the policy that allows the highest amount of change, `Min` to use the policy that allows the lowest amount of change, or `Disabled` to prevent the scaling policy from scaling in that policy direction. The default value is `Max`.
+<6> The `stabilizationWindow` parameter determines the time period the HPA should look back at desired states. The default value is `0`.
+
+For example, the following example shows a scaling policy for scaling down:
+[source,yaml]
+----
+spec:
+  behavior:
+    scaleDown:
+      stabilizationWindowSeconds: 300
+      policies:
+      - type: Pods
+        value: 4
+        periodSeconds: 60
+      - type: Percent
+        value: 20
+        periodSeconds: 60
+      selectPolicy: Max
+...
+  minReplicas: 20
+...
+----
+
+When the number of pods is more than 40, the percent-based policy is used for scaling down, as the HPA determines that policy results in a larger change, as required by the `selectPolicy`. 
```

Comment:
So on this one, it would only take 20 pods (not 40) to make the percent one bigger. For example, if there were 25 pods, 20% of 25 pods is 5 pods, so since 5 is bigger than 4 (as specified in the `Pods` policy), then the `Percent` would be used. Did you maybe mean to say 10% instead of 20%? If you keep 20%, you'll also need to update the paragraph below.

Reply:
@mburke5678 it seems joel's comments did not get addressed -- 20 pods (not 40)

Reply:
@weinliu Thank you for catching this. However, i used Joel's suggestion of keeping the 40 pods and updated the paragraph to use the proper calculations. 
If you could be so kinf as to verify, I would greatly appreciate  it: https://github.com/openshift/openshift-docs/pull/23557/files#diff-3061c40d7af94783585213707ba8933f331ff8ec77049a856600e4ee412b22e5R77